roles/confluent.ssl/tasks/custom_certs.yml

```diff
@@ -27,15 +42,20 @@
     keytool -noprompt -keystore {{truststore_path}} \
       -storetype pkcs12 \
       -alias CARoot \
-      -import -file /var/ssl/private/generation/{{ssl_ca_cert_filepath|basename}} \
+      -import -file {{ssl_ca_cert}} \
       -storepass {{truststore_storepass}} \
       -keypass {{truststore_storepass}} {{extra_args}}
+  when: not ssl_add_ca_chain
```

Comment:
might be nice to allow the code to decide if its a ca bundle vs a single cert, would look something like this
```
- name: Count certs
  shell: |
    grep -c "END CERTIFICATE" /var/ssl/private/generation/{{ssl_ca_cert_filepath|basename}}
  register: cert_count_grep

- name: Set the Certificate Count Var
  set_fact:
      cert_count: "{{cert_count_grep.stdout | int}}

- name: Create Truststore and Import the CA Chain
  include_tasks: import_ca_chain.yml
  when: cert_count>1
```

Reply:
this would eliminate the need for another variable

Reply:
Ok cool. I just thought I’d leave the original task in there.